cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/image/AzureImageIdService.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.cloudbreak.cloud.azure.image;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.util.Assert;
+
+import com.sequenceiq.cloudbreak.cloud.azure.AzureStorage;
+
+@Component
+public class AzureImageIdService {
```

Comment:
I would call it AzureImageIdProviderService - and rename it later to ImageIdService once there are also other methods